src/App.css

```diff
@@ -1,39 +1,299 @@
+
+:root {
+  --monsterat: 'Montserrat', sans-serif;
+  --robotoSlab: 'Roboto Slab', serif;
+  --bubble-gum-pink: #ff5ab7;
+  --black: #1e1e1e;
+  --medium-blue: #307bbe;
+  --white: #fafafa;
+  --pinkish-grey: #c4c4c4;
+  --pale-grey: #f5f6fa;
+  --dodger-blue: #379cf6;
+  --azure: #0290ff;
+  --warm-grey: #888;
+  --cool-blue: #4386bf;
+  --black-5: rgba(0, 0, 0, 0.05);
+  --white-two: #fff;
+  --black-two: #121212;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.container {
+  margin: 0 auto;
+  background-color: var(--white);
+  overflow-x: hidden;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.navbar {
+  display: flex;
+  height: 95px;
+  background-color: var(--white-two);
+  align-items: center;
+  margin-bottom: 20px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.navcontainer {
   display: flex;
-  flex-direction: column;
+  margin: 0 auto;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
+  width: 90vw;
+}
+
+.logo h1 {
+  font-family: var(--monsterat);
+  color: var(--azure);
+}
+
+.catSelect {
+  position: absolute;
+  top: 30px;
+  left: 45vw;
+  opacity: 0.2;
+  padding: 10px 20px;
+}
+
+.catSelect:focus {
+  opacity: 1;
+}
+
+.navigation {
+  display: flex;
+  list-style: none;
+}
+
+.navigation li {
+  text-transform: uppercase;
+  margin-left: 20px;
+  font-family: var(--black-two);
+}
+
+.userIcon {
+  width: 45px;
+  height: 45px;
+  border-radius: 50%;
+  border: solid 2px #e8e8e8;
+  margin-left: auto;
+  position: relative;
+}
+
+.svgcont {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-31%, -32%);
+}
+
+.userSvg {
+  fill: var(--azure);
+  width: 32px;
+  height: 32px;
+}
+
+.bookContainer {
+  width: 90vw;
+  margin: 0 auto 40px auto;
+}
+
+.book {
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: var(--white-two);
+  padding: 32px 27px 26px 27px;
+  margin-bottom: 15px;
+  display: flex;
+  height: 170px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.bookInfo {
+  display: flex;
+  flex-direction: column;
+}
+
+.catTitAuth {
+  display: flex;
+  flex-direction: column;
+}
+
+.catTitAuth p {
+  margin: 0 39px 0 0;
+  opacity: 0.5;
+  font-family: var(--monsterat);
+  font-size: 14px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.catTitAuth h4 {
+  font-family: var(--robotoSlab);
+  font-size: 14px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--cool-blue);
+}
+
+.catTitAuth h2 {
+  font-family: var(--robotoSlab);
+  font-size: 22px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
```

Comment:
## Aim to Stay `DRY`
- [ ] There are some repeated rules in these classes. Do you mind merging these rules as they appear similar so as to not appear like you are repeating yourself? We could create a common class for this. What do you think?